test/jdk/javax/swing/JColorChooser/Test4887836.java

```diff
@@ -21,23 +21,42 @@
  * questions.
  */
 
+import java.awt.Color;
+import java.awt.Font;
+import javax.swing.JColorChooser;
+import javax.swing.UIManager;
+
 /*
  * @test
  * @bug 4887836
- * @summary Checks if no tooltip modification when no KeyStroke modifier
- * @author Konstantin Eremin
- * @run applet/manual=yesno Test4887836.html
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Checks for white area under the JColorChooser Swatch tab
+ * @run main/manual Test4887836
  */
 
-import java.awt.Color;
-import java.awt.Font;
-import javax.swing.JApplet;
-import javax.swing.JColorChooser;
-import javax.swing.UIManager;
+public class Test4887836 {
+
+    public static void main(String[] args) throws Exception {
+        String instructions = """
+                                If you do not see white area under the \"Swatches\" tab,\n +
+                                then test passed, otherwise it failed.""";
```

Comment:
please fix line-lengths and the "+" operator is not required since String blocks are used.

Reply:
Please fix line-lengths.